aesara/tensor/math_opt.py

```diff
@@ -3052,7 +3052,8 @@ def _is_1(expr):
 register_stabilize(logsigm_to_softplus, name="logsigm_to_softplus")
 register_stabilize(log1msigm_to_softplus, name="log1msigm_to_softplus")
 register_stabilize(log1pexp_to_softplus, name="log1pexp_to_softplus")
-register_stabilize(log1p_neg_sigmoid, name="log1p_neg_sigmoid,")
+register_stabilize(log1p_neg_sigmoid, name="log1p_neg_sigmoid")
+register_specialize(log1p_neg_sigmoid, name="log1p_neg_sigmoid")
```

Comment:
This one was actually mentioned in a previous discussion here: https://github.com/aesara-devs/aesara/issues/29#issuecomment-680818970